lib/utils/axios.ts

```diff
@@ -20,6 +21,9 @@ export function getAxiosClient(
 
             return axiosRetry.isNetworkError(error) || axiosRetry.isRetryableError(error);
         },
+        onRetry: (retryCount, error, requestConfig) => {
+            _.set(requestConfig, ['headers', 'x-request-attempt'], retryCount || 0);
```

Comment:
may be there should be retryCount || 1 ?
because we are already inside onRetry so this is the first retry

Reply:
Yeah, good idea